docs/tutorials/brownian_motion.py

```diff
@@ -0,0 +1,96 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: .venv
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Brownian Motion
+#
+# Brownian motion describes motion of small particles with stochastic forces applied to them. The math of Brownian motion can be modeled with Wiener process. In this tutorial, we take a simple form of the model and treat the stochastic forces as Gaussian.
+#
+# For consistency, we always use $\mathbf x$ for displacement, and $t$ for steps. The model we are using is
+#
+# $$
+# \begin{align}
+# \mathbf x(t + \mathrm dt) &= \mathrm x(t) + \mathcal{N}(\mu=0, \sigma=\sigma \sqrt{\mathrm d t})
```

Comment:
- left-hand side $\mathbf{x}$ looks like a US-styled vector; but right-hand side $\mathrm{x}$ looks like a scalar.
- $\sqrt{\mathrm{d} t}$ looks mathematically awkward

Reply:
1. typo, fixed in cc77bb2 , also fixed in the docstrings.
2. hmm not sure what you mean.